app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,105 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>A modern languages teacher can  be a rewarding way to use your language knowledge, while opening young minds to the languages and cultures around the world.</p>
```

Comment:
@caitlin-reidy this first sentence doesn't quite make sense. Think it needs to say Being a modern languages teacher... or something similar? Sentence has quite a few language/s in it. Can we delete the bit about using your language knowledge and just say Being a modern languages teacher is rewarding as you can open young minds...